packages/terra-table/src/proptypes/sectionShape.js

```diff
@@ -22,6 +23,10 @@ const sectionShape = PropTypes.shape({
    * An array of row objects to be rendered within the section.
    */
   rows: PropTypes.arrayOf(rowShape),
+  /**
+   * An array of subsections objects to be rendered within the section.
+   */
+  subsections: PropTypes.arrayOf(subsectionShape),
```

Comment:
A documentation update is needed per subsection property added to the section shape here:
https://github.com/cerner/terra-framework/blob/cc25304e3fcaa751253ae37f6763f763566b1b44/packages/terra-framework-docs/src/terra-dev-site/doc/table/About.1.doc.mdx?plain=1#L148